docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  web:
+    build: .
+    ports:
+     - "8080:80"
+    volumes:
+      - ./app.conf:/etc/nginx/conf.d/app.conf
+      - ./index.php:/var/www/html/index.php
+      - ./setVersion.sh:/usr/local/bin/setVersion.sh
+      - ./nginxVersion.env:/var/nginxVersion.env
+    command: >
+      /bin/bash -c "/usr/local/bin/setVersion.sh
```

Comment:
Hey, you made it in a way I didn't think about. Originally I thought that `VERSION` variable is set during build :)
I will update task.

I think you can close your PR and proceed on to the next task

Reply:
Helped me get my head around .env files and get a little better with Bash.

Is it possible to do this within the container without restarting? or would have its own security implications, allowing containers to set their own env-vars?